bolt/test/X86/infer_no_exits.test

```diff
@@ -1,8 +1,10 @@
 ## This verifies that functions where an exit block has a landing pad are covered by stale profile inference.
-# RUN: %clangxx %cxxflags %p/Inputs/infer_no_exits.cpp -o %t.exe
+# RUN: %clangxx %cxxflags %p/Inputs/infer_no_exits.s -o %t.exe
 # RUN: link_fdata %s %t.exe %t.preagg PREAGG
 # RUN: perf2bolt %t.exe -p %t.preagg --pa -o %t.fdata -w %t.yaml
 # RUN: sed -i 's/0000/0001/g' %t.yaml
+# RUN: sed -i 's/hash:            0x9AE3FDBFFBE36274/hash:            0x0000000000000000/g' %t.yaml
```

Comment:
Let's make it robust wrt possible BF hash changes: match the line with BF hash and replace the hash with 0x0000000000000000.